awswrangler/s3/_read_parquet.py

```diff
@@ -506,6 +507,7 @@ def _read_parquet(
 
 def read_parquet(
```

Comment:
Should we be doing this for `read_text` methods (e.g. `read_csv`) as well?

Reply:
Do you mean the same partition optimisation? We're missing a few key methods for others at this point like `read_csv_table` (that's when `path_root` really becomes useful), but ideally yes.